package.json

```diff
@@ -13,6 +24,19 @@
     "express": "~4.15.2",
     "mongoose": "^4.10.8",
     "morgan": "~1.8.1",
+    "nodemon": "^1.11.0",
```

Comment:
Nodemon is in the wrong place. It is a Dev Dependency as it wont be used on the final build server (Heroku). Also since you are adding it why not use it in the start command, or even better define a custom start:dev command.